parlai/agents/transformer/modules.py

```diff
@@ -194,7 +214,7 @@ def __init__(self, transformer, hdim):
         self.transformer = transformer
         self.mlp = nn.Sequential(
             nn.Linear(dim, hdim),
-            nn.ReLU(),
+            nn.ReLU(),  # TODO: should this also be gelu?
```

Comment:
@emilydinan  opinions?

Reply:
hmmm... good question. would have to try some experiments, but maybe we can pass teh activation in as an arg